playbook-website/app/javascript/components/Website/index.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState, useEffect } from "react"
+import { Layout, Icon } from "playbook-ui"
+import Sidebar from "./src/layouts/Sidebar"
+import LayoutRight from "./src/layouts/LayoutRight"
+
+function App() {
+  const [kits, setKits] = useState([])
+  const [dark, setDark] = useState(false)
+  const [isMobile, setIsMobile] = useState(window.innerWidth < 768)
+
+  useEffect(() => {
+    fetch("/beta/kits.json")
```

Comment:
This `fetch` seems a bit coupled here with the view itself. My suggestion would be to abstract any `fetch` calls out to an API layer.